OpenSim/Simulation/Model/Model.h

```diff
@@ -943,6 +943,14 @@ OpenSim_OBJECT_NONTEMPLATE_DEFS(Model, ModelComponent);
     std::vector<SimTK::ReferencePtr<const OpenSim::Coordinate>>
         getCoordinatesInMultibodyTreeOrder() const;
 
+    /** A variant of getCoordinatesInMultibodyTreeOrder that returns names for Scripting users */
+    SimTK::Array_<std::string> getCoordinateNamesInMultibodyTreeOrder() {
+        SimTK::Array_<std::string> namesArray;
```

Comment:
Any reason for using a `SimTK::Array` over an `OpenSim::Array` or a `std::vector`?

Reply:
OpenSim::Array has been disfavored for a while in the hope of retiring it, but the choice between SimTK::Array and std::vector is a bit arbitrary since both have already been exposed across the interface and are used in the same file and extensively across the API.